src/nimibook/resources/templates/document.mustache

```diff
@@ -58,7 +58,7 @@
     <body>
         <!-- Provide site root to javascript -->
         <script type="text/javascript">
-            var path_to_root = "{{ path_to_root }}";
+            var path_to_root = "{{ path_to_root }}/assets";
```

Comment:
I guess whether or not this is correct depends on how the other js scripts (which I have not looked at) use and interpret the path_to_root? if it is only to interact with assets this is correct. If it is only to interact with the pages, the original is not correct. If they use it for both uses, we might have to reconsider the use of assets folder inside docs (or patch the js scripts)

Reply:
A quick grep in the resources folder shows that ``path_to_root`` is only used ``document.mustache``  so the js scripts should not use ``path_to_root``, I guess ?

To the best of my understanding (which is not great - javascript is not my friend), ``path_to_root`` is only used to call java script in the generated html.

Reply:
yeah, it seems a false alarm. ok for me.